packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -4,7 +4,7 @@ import 'package:provider/provider.dart';
 import 'package:qr_code_scanner/qr_code_scanner.dart';
 import 'package:smooth_app/data_models/continuous_scan_model.dart';
 import 'package:smooth_app/pages/personalized_ranking_page.dart';
-import 'package:smooth_app/pages/smooth_bottom_navigation_bar.dart';
+import 'package:smooth_app/pages/smooth_bottom_navigation_bar.dart' as navbar;
```

Comment:
It was because for some reason the compiler complained that `SmoothBottomNavigationBar` was defined in two files - I had no clue what caused it by this fixed it.

Now I can't repro that problem, so I removed the fix :)